scratchpad/scratchpad

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env bash
+
+# Nikhil Singh <nik.singh710@gmail.com>
+
+# Variables
+_listing=false
+_menu_cmd=("rofi" "-dmenu" "-i")
+green="\033[0;32m"
+red="\033[0;31m"
+blue="\033[0;34m"
+nocolor="\033[0m"
+
+check() {
+	command -v "$1" 1>/dev/null
+}
+
+ok() {
+	echo -e "[$green  $nocolor] $*"
+}
+
+err() {
+	echo -e "[$red  $nocolor] $*"
+}
+
+optional() {
+	echo -e "[$blue  $nocolor] $*"
+}
+
+notify() {
+	check notify-send && {
+		notify-send "$@"
+	} || {
+		echo "$@"
+	}
+}
+
+checkUtils() {
+	check sed && ok "sed" || err "sed"
+	check jq && ok "jq" || err "jq"
+	check notify-send && ok "notify-send (Optional)" || optional "notify-send (Optional)"
+	exit
+}
+
+basicChecks() {
+	check hyprctl || {
+		notify "Seriously mate!!" "Use Hyprland then this script"
+		exit 1
+	}
+	pgrep -x Hyprland &>/dev/null || {
+		notify "Make Sure" "Hyprland Session is running."
```

Comment:
I'm not sure having this sentence separated as title and body is ideal. Could you write it all as the title?

Reply:
Yes, I can I guess I liked it this way because of my notification md configuration.